src/main/java/org/apache/commons/imaging/formats/gif/GifImageMetadataItem.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.imaging.formats.gif;
+
+import org.apache.commons.imaging.common.ImageMetadata;
+
+public class GifImageMetadataItem implements ImageMetadata.ImageMetadataItem {
+    private static final String NEWLINE = System.getProperty("line.separator");
+    private final int delay;
+    private final int leftPosition;
+    private final int topPosition;
```

Comment:
Maybe we should include the `width` and `height` of each gif image?

Reply:
If possible. Just tried changing the code a bit, and I couldn't find the data for each image. Wondering if all we get is the left/top values instead?

Reply:
I only use the metadata to decide the delay and placement for each image. For that I need the overall GIF width and height as well as the individual image offsets. As you say, you can easily get the width and height from each buffered image. Maybe it could be useful if you want to display such information before loading the actual images?

Reply:
Either we add it now just in case or we wait until we see there is an actual need for it? I'll let you decide! :) I can fix it if needed tomorrow.

Reply:
Let's get this one merged in then, and add more later if necessary. Thanks!